forge-gui/res/editions/Mystery Booster Playtest Cards.txt

```diff
@@ -68,7 +68,7 @@ ScryfallCode=CMB1
 59 R Planequake @Chris Kiritz
 60 R Problematic Volcano @Gavin Verhey
 61 R Queue of Beetles @Brendan Sell
-62 R Red Herring @Chris Mooney
+62 R Red Herring (Playtest) @Chris Mooney
```

Comment:
@paulsnoops is this right?

Reply:
No, (Playtest) shouldn't be there as it's not actually part of the name. Although I see why you've added it as there are now two different cards with this name, gg Wizards. Maybe we just comment it out for now as this card isn't in Forge at the moment.

Reply:
@tehdiplomat had a good suggestion that this card should be listed instead as P-Red Herring